src/hoi4_world/states/hoi4_states_converter.cpp

```diff
@@ -0,0 +1,117 @@
+#include "src/hoi4_world/states/hoi4_states_converter.h"
+
+#include <algorithm>
+#include <vector>
+
+#include "external/commonItems/Log.h"
+#include "external/fmt/include/fmt/format.h"
+
+
+
+namespace
+{
```

Comment:
What's the deal with partial unnamed namespace? Hiding these functions from the class interface?

Reply:
It's the preferred method for making them non-global, so that an extern somewhere else can't grab what's meant to be a local function.